praetorian_cli/sdk/test/ui/test_aegis_ssh.py

```
@@ -0,0 +1,60 @@
+
+
+def test_handle_ssh_user_extraction_and_strip_l_flag():
+    menu = Menu()
+    args = ["-u", "admin", "-L", "8080:localhost:80", "-l", "ignored"]
+    handle_ssh(menu, args)
+
+    assert len(menu.sdk.aegis.calls) == 1
+    call = menu.sdk.aegis.calls[0]
+    assert call['user'] == "admin"
+    assert "-l" not in call['options'] and "ignored" not in call['options']
+    assert ["-L", "8080:localhost:80"] == call['options']
```

Ruff (0.12.2)

Yoda condition detected

Rewrite as `call['options'] == ["-L", "8080:localhost:80"]`

(SIM300)